src/Lykke.Service.BlockchainBalances.Services/BlockchainBalancesService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using System.Numerics;
+using System.Threading.Tasks;
+using Flurl;
+using Flurl.Http;
+using Lykke.Service.BlockchainBalances.Core.Domain;
+using Lykke.Service.BlockchainBalances.Core.Services;
+using Lykke.Service.BlockchainBalances.Core.Settings.ClientSettings;
+using Nethereum.Util;
+using BalanceModel = Lykke.Service.BlockchainBalances.Core.Domain.BalanceModel;
+
+namespace Lykke.Service.BlockchainBalances.Services
+{
+    public class BlockchainBalancesService : IBlockchainBalanceService
+    {
+        private const decimal BtcMultiplier = 0.00000001m;
+        private readonly string _ninjaUrl;
+        private readonly string _samuraiUrl;
+
+        public BlockchainBalancesService(
+            NinjaServcieSettings ninjaServiceSettings,
+            EtheriumSamuraiServiceSettings samuraiServiceSettings
+            )
+        {
+            _ninjaUrl = ninjaServiceSettings.ServiceUrl;
+            _samuraiUrl = samuraiServiceSettings.ServiceUrl;
+        }
+
+        public async Task<BalanceModel> GetBalanceAsync(string address, Blockchain blockchain)
```

Comment:
What if balance for specified parameters is not found?

Reply:
Balance property of the BalanceModel will be 0